tools/lint-docstring

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+
+import argparse
+import importlib
+import sys
+from os import getcwd, listdir
+from os.path import isfile, join
+from typing import Dict, List, Tuple, cast
+
+
+developer_doc = join(getcwd(), 'docs', 'developer-file-overview.md')
+
+
+def get_docstrings() -> Tuple[Dict[str, str], Dict[str, List[str]]]:
```

Comment:
This is specifically *module* docstrings, so it would be clearer to name it as such?